README.md

```diff
@@ -91,11 +91,9 @@ web:
 5. provision vagrant host using:
 
 ```bash
-# Run the full playbook
-ansible-playbook playbook.predeploy.yml --private-key=.vagrant/machines/default/virtualbox/private_key -u vagrant -i inventory/vagrant --extra-vars="config_file=./vars/example-vars-staging.yml"
-
-# Limit to just the tasks for the web servers
-ansible-playbook playbook.predeploy.yml --private-key=.vagrant/machines/default/virtualbox/private_key -u vagrant -i inventory/vagrant --extra-vars="config_file=./vars/example-vars-staging.yml" -l web
+# Run the vagrant provisioner to create the db_deploy_users
+playbook=playbook.vagrant.yml config_file=appname_expanded_vars.yml vagrant provision
 
+# Run the full provisioner
+playbook=playbook.predeploy.yml config_file=appname_expanded_vars.yml vagrant provision
```

Comment:
If setting the environment variables is preferred, we should use all caps for the environment variable names.

I would prefer to have the vagrant example resemble the actual remote host use case.  When actually being run against a target set of hosts, we won't be using a Vagrant file.